Automattic/wp-calypso
# Refactor: Remove isMarketplaceProduct selector when not required

The isMarketplaceProduct flag is already in the plugin object

#### Proposed Changes

* TBD

#### Testing Instructions


Make sure the following scenarios work identically in Production
* The eligibility warnings dialog when purchasing a plugin on a site that is Business or greater but it is not yet Atomic
* Clicking the CTA button when installing a Plugin
* The Remove button in the Purchases screen
* Plugin browser list shows the same data

#### Pre-merge Checklist






Related to #
